snapshots/testsuite/testsuite.go

```diff
@@ -18,8 +18,6 @@ package testsuite
 
 import (
 	"context"
-	//nolint:golint
-	_ "crypto/sha256"
```

Comment:
This was not needed? I see it was added in https://github.com/containerd/containerd/commit/027ee569a3bcebed1f55f0aabdb5c0db35fc79cf

And I recently ran into something similar (see https://github.com/opencontainers/go-digest/pull/64)

/cc @estesp